packages/wdio-cucumber-framework/src/wdioEventTranslator.js

```diff
@@ -0,0 +1,298 @@
+import { Status } from 'cucumber'
+import { CucumberEventListener } from './eventListener'
+import { createStepArgument } from './utils'
+import * as path from 'path'
+import logger from '@wdio/logger'
+
+const SETTLE_TIMEOUT = 5000
+const log = logger('wdio-cucumber-framework:CucumberWdioEventTranslator')
+
+/**
+ * CucumberReporter
+ * responsible for accepting the cucumber specific events emitted by CucumberEventListener
+ * and emitting the standard WebdriverIO test runtime events
+ * The translation is:
+ *
+ * before-feature/after-feature -> suite:start/suite:end
```

Comment:
Should we document if this diverges from the previous link between Cucumber and WDIO

Reply:
Good thought. These translations are the same as we had before so I don't think we need any new documentation.

Reply:
👍